### PR TITLE
Minor things noticed while merging to 1.10

### PR DESCRIPTION
**Pull request by derobins**

### CODE REVIEW

File: test/vfd.c [hunk 1]
```diff
@@ -5975,7 +5975,7 @@ main(void)
      */
     env_h5_drvr = HDgetenv(HDF5_DRIVER);
     if (env_h5_drvr) {
-        HDprintf(" -- SKIPPED VFD tests because %s is set -- \n", HDF5_DRIVER);
+        HDprintf(" -- SKIPPED VFD tests because %s is set -- \n", env_h5_drvr);
```
Comment (derobins) [inline test/vfd.c]: Probably want the actual environment variable string instead of "HDF5_DRIVER" here.

**Comment by derobins** [comment]
A lot of this was just fixing mixed tabs and spaces

File: test/vfd.c (hunk 1 above)
Comment (jhendersonHDF) [reply test/vfd.c]: HDF5_DRIVER IS the actual environment string. The message was meant to tell the user that "HDF5_DRIVER" is set, not what the value was

**Review by jhendersonHDF — CHANGES_REQUESTED** [review]
HDF5_DRIVER is a macro that was added which maps to "HDF5_DRIVER" in case it needs to be changed in the future.

**Comment by derobins** [comment]
> HDF5_DRIVER is a macro that was added which maps to "HDF5_DRIVER" in case it needs to be changed in the future.

Ah, I see what you were going for. I'll change that back.

File: test/vfd.c (hunk 1 above)
Comment (derobins) [reply test/vfd.c]: Fixed

**Review by jhendersonHDF — APPROVED** [review]

**Review by lrknox — APPROVED** [review]

**Pull request merged**